detectors/TransLab/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM pytorch/pytorch:2.0.0-cuda11.7-cudnn8-runtime
+
+ARG DETECTOR_DIR=TransLab
+
+WORKDIR /detector
+
+COPY ../common common
```

Comment:
Parent directory cannot be accessed in docker. [See](https://docs.docker.com/build/building/context/)